rotkehlchen/tests/unit/test_bitcoin.py

```diff
@@ -1,5 +1,8 @@
+from unittest.mock import MagicMock, patch
```

Comment:
I don't think it is a good idea to mix `unittest` with `pytest`. As far as I know they are different and unrelated frameworks

Reply:
Hmmm, yes, but we're writing unittests here, so I'll have to use it to mock the failure? I don't know if there's another way to do this, do you have any idea?

Reply:
No no it's fine. We use magic mocks from unittest all over the code. Pytest and unittest complement each other. It's fine.